src/librustc_data_structures/stable_map.rs

```diff
@@ -0,0 +1,80 @@
+pub use rustc_hash::FxHashMap;
+use std::borrow::Borrow;
+use std::collections::hash_map::{Entry, IterMut};
+use std::hash::Hash;
+
+/// A deterministic wrapper around FxHashMap that does not provide iteration support.
+///
+/// It supports insert, remove, get and get_mut functions from FxHashMap.
+/// It also allows to convert hashmap to a sorted vector with the function `into_sorted_vector()`.
+#[derive(Debug, Clone, Default, PartialEq, Eq)]
+pub struct StableMap<K, V>
+where
+    K: Eq + Hash,
+{
+    base: FxHashMap<K, V>,
+}
+
+impl<K: Hash + Eq + Ord, V: Ord> StableMap<K, V> {
```

Comment:
Please dont require `K: Ord` and `V: Ord`.

Reply:
OK. Thanks.